localisation/000_wc_military_localization.csv

```diff
@@ -0,0 +1,455 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+#Military Technologies;;;;;;;;;;;;;;x
+TECH_INFANRY;Conventional Warfare;;;;;;;;;;;;;x
+TECH_CAVALRY;Irregular Warfare;;;;;;;;;;;;;x
+TECH_SKIRMISH;Ley Attunement;;;;;;;;;;;;;x
+TECH_MELEE;Tactics Development;;;;;;;;;;;;;x
+TECH_SIEGE_EQUIPMENT;Military Engineering;;;;;;;;;;;;;x
+TECH_RECRUITMENT;Martial Logistics;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+#ABBREVIATIONS;;;;;;;;;;;;;;x
+L_INF;CA;;;;;;;;;;;;;x
+H_INF;IN;;;;;;;;;;;;;x
+PIKE;AI;;;;;;;;;;;;;x
+L_CAV;CV;;;;;;;;;;;;;x
+ARCH;RI;;;;;;;;;;;;;x
+H_CAV;LI;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+KNIGHTS;Large Infantry;;;;;;;;;;;;;x
+KNIGHTS_OFFENSIVE_FIRE;Large Infantry Attack;;;;;;;;;;;;;x
+KNIGHTS_DEFENSIVE_FIRE;Large Infantry Defense;;;;;;;;;;;;;x
+KNIGHTS_MORALE;Large Infantry Morale;;;;;;;;;;;;;x
+KNIGHTS_MODIFIER;Large Infantry Modifier;;;;;;;;;;;;;x
+CU_KNIGHTS;Large Infantry;;;;;;;;;;;;;x
+CU_HEAVY_CAVALRY;Large Infantry;;;;;;;;;;;;;x
+PIKEMEN;Aerial Infantry;;;;;;;;;;;;;x
+PIKEMEN_OFFENSIVE_FIRE;Aerial Infantry Attack;;;;;;;;;;;;;x
+PIKEMEN_DEFENSIVE_FIRE;Aerial Infantry Defense;;;;;;;;;;;;;x
+PIKEMEN_MORALE;Aerial Infantry Morale;;;;;;;;;;;;;x
+PIKEMEN_MODIFIER;Aerial Infantry Modifier;;;;;;;;;;;;;x
+CU_PIKEMEN;Aerial Infantry;;;;;;;;;;;;;x
+LIGHT_CAVALRY;Cavalry;;;;;;;;;;;;;x
+LIGHT_CAVALRY_OFFENSIVE_FIRE;Cavalry Attack;;;;;;;;;;;;;x
+LIGHT_CAVALRY_DEFENSIVE_FIRE;Cavalry Defense;;;;;;;;;;;;;x
+LIGHT_CAVALRY_MORALE;Cavalry Morale;;;;;;;;;;;;;x
+LIGHT_CAVALRY_MODIFIER;Cavalry Modifier;;;;;;;;;;;;;x
+CU_LIGHT_CAVALRY;Cavalry;;;;;;;;;;;;;x
+ARCHERS;Ranged Infantry;;;;;;;;;;;;;x
+ARCHERS_OFFENSIVE_FIRE;Ranged Infantry Attack;;;;;;;;;;;;;x
+ARCHERS_DEFENSIVE_FIRE;Ranged Infantry Defense;;;;;;;;;;;;;x
+ARCHERS_MORALE;Ranged Infantry Morale;;;;;;;;;;;;;x
+ARCHERS_MODIFIER;Ranged Infantry Modifier;;;;;;;;;;;;;x
+CU_ARCHERS;Ranged Infantry;;;;;;;;;;;;;x
+HEAVY_INFANTRY;Infantry;;;;;;;;;;;;;x
+HEAVY_INFANTRY_OFFENSIVE_FIRE;Infantry Attack;;;;;;;;;;;;;x
+HEAVY_INFANTRY_DEFENSIVE_FIRE;Infantry Defense;;;;;;;;;;;;;x
+HEAVY_INFANTRY_MORALE;Infantry Morale;;;;;;;;;;;;;x
+HEAVY_INFANTRY_MODIFIER;Infantry Modifier;;;;;;;;;;;;;x
+CU_HEAVY_INFANTRY;Infantry;;;;;;;;;;;;;x
+CU_ACTUAL_HEAVY_INFANTRY;Infantrymen;;;;;;;;;;;;;
+LIGHT_INFANTRY;Casters;;;;;;;;;;;;;x
+LIGHT_INFANTRY_OFFENSIVE_FIRE;Casters Attack;;;;;;;;;;;;;x
+LIGHT_INFANTRY_DEFENSIVE_FIRE;Casters Defense;;;;;;;;;;;;;x
+LIGHT_INFANTRY_MORALE;Casters Morale;;;;;;;;;;;;;x
+LIGHT_INFANTRY_MODIFIER;Casters Modifier;;;;;;;;;;;;;x
+CU_LIGHT_INFANTRY;Casters;;;;;;;;;;;;;x
+CU_ACTUAL_LIGHT_INFANTRY;Spellcasters;;;;;;;;;;;;;
```

Comment:
If there are both "actual" and "not actual" units, then "actual" is actual units and "not actual" is category.
So, casters and ranged Infantry are both in "Casters" category and Aerial Units and Infantry and both in "Infantry" category.